terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/config/PropertySourceConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.terasoluna.gfw.functionaltest.config;

```

Comment:
I add copyright PropertySourceConfig.java at f439f8460c7d7e0ccfd4eb710b0c507240341724 commit.
and I realized to forget adding copyright at ApServerName.java ...
so I add copyright at ApServerName.java at dbc894485791b4777f8cf25f0c2bc0d3ff96a3d6 commit